requirements.txt

```diff
@@ -10,3 +10,4 @@ Yapsy==1.11.223
 configobj==4.7.0
 redis>=2.10.5
 morph==0.1.2
+ibm_db>=2.0.7
```

Comment:
This way we keep on adding dependencies to crawler even when a plugin (redis, db2) may not be used in a given environment.
Do you think something like `pip.main(['install', 'ibm_db'])` would work without conflicts?

http://stackoverflow.com/questions/17271444/how-to-install-a-missing-python-package-from-inside-the-script-that-needs-it
http://stackoverflow.com/questions/12332975/installing-python-module-within-code